app/src/main/res/values/strings.xml

```diff
@@ -149,13 +149,15 @@
 
     <!--Seed-->
     <string name="save_seed_instruction">Write down this seed phrase and save it in a secure location. You will be asked to enter it on the next screen and if you ever need to restore your wallet.</string>
-    <string name="please_confirm_your_seed_by_typing_and_tapping_each_word_accordingly">Please confirm your seed by typing and tapping each word accordingly</string>
+    <string name="please_confirm_your_seed_by_typing_and_tapping_each_word_accordingly">Please, confirm your seed by typing and tapping each word accordingly</string>
+    <string name="please_select_correct_seed">Please, confirm your seed by selecting each correct word accordingly</string>
     <string name="enter_your_seed">Enter your seed…</string>
+    <string name="theInputFieldIsEmpty">The input fields is empty. Please, enter the seeds.</string>
     <string name="error_seed_not_generated">Error occurred, Seed was not generated</string>
     <string name="warning_save_seed">Failure to keep this seed safe can result in theft of your entire wallet or inability to recover your wallet</string>
-    <string name="tap_to_select">Tap to select Word</string>
-    <string name="incorrect_seed_input">You entered an incorrect seed, Please confirm your seed</string>
-    <string name="i_copied_the_seed_phrase">I copied the seed phrases</string>
+    <string name="tap_to_select">Tap to select the word</string>
+    <string name="incorrect_seed_input">You entered an incorrect seed, please check your seed</string>
```

Comment:
Error message could be changed to "You selected an incorrect seed, please check your seed" since we're no longer entering the seed phrases with keyboard. You could use a different error for restore wallet.

Reply:
Fixed in commit: "confirm_seed: strings formating"